diff --git a/src/Plugin/Deriver/TypedDataRelationshipDeriver.php b/src/Plugin/Deriver/TypedDataRelationshipDeriver.php
index 608f5db5..cd3908e9 100644
--- a/src/Plugin/Deriver/TypedDataRelationshipDeriver.php
+++ b/src/Plugin/Deriver/TypedDataRelationshipDeriver.php
@@ -38,10 +38,10 @@ class TypedDataRelationshipDeriver extends TypedDataPropertyDeriverBase implemen
         $derivative['data_type'] = $main_property->getDataType();
         $derivative['property_name'] = $property_name;
         if (strpos($base_data_type, 'entity:') === 0) {
-          $context_definition = new EntityContextDefinition($base_data_type, $this->typedDataManager->createDataDefinition($base_data_type));
+          $context_definition = new EntityContextDefinition($base_data_type, 'Base');
         }
         else {
-          $context_definition = new ContextDefinition($base_data_type, $this->typedDataManager->createDataDefinition($base_data_type));
+          $context_definition = new ContextDefinition($base_data_type, 'Base');
         }
         // Add the constraints of the base definition to the context definition.
         if ($base_definition->getConstraint('Bundle')) {
